render.yaml

```diff
@@ -0,0 +1,225 @@
+# Configure deployment on Render.com infrastructure.
+# For info on options, see: https://render.com/docs/blueprint-spec
+#
+# THIS IS AN EXPERIMENT. Our primary deployment is still on AWS, but we hope to
+# switch to this if it works well and is not too expensive. See:
+# https://github.com/usdigitalresponse/univaf/issues/683
+
+databases:
+  - name: univaf-db-production
+    plan: pro
```

Comment:
This plan is a little smaller than what we are using on RDS right now. Render’s `pro` is:
- 2 CPUs
- 4 GB RAM
- 96 GB of disk
- <= 97 connections

On RDS, we are using a [`db.m5.large` instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html):
- 2 CPUs
- 8 GB RAM
- No imposed connection limit, but we average around 30 and occasionally spike to around 92.
- We have 250 GB of disk but only need ~30 GB; the overage is from back when the DB was growing out of control and we never shrunk after fixing the issues.

We typically have ~5 GB of freeable memory and ~25% CPU utilization (with spikes every 6 hours to ~60%; this is probably autovacuuming and shouldn’t be a real issue). It *seems* to me that this DB plan is probably all we actually need, but if there are issues we can boost it to `pro plus` (4 CPUs, 8 GB RAM, 256 GB disk, 197 connections) → $95/month vs. $185/month.

Last month we paid $158.48 on RDS ($2.72 for backups, $128.16 for the instance, $27.60 for the disk).

Reply:
<img width="990" alt="render-db-plans" src="https://user-images.githubusercontent.com/74178/178359569-ab9e8950-1fc6-4cc0-a15a-59f35e45ce09.png">